go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/greenbone/go-project-template
 
-go 1.16
+go 1.18
```

Comment:
ah cool - I just commented in https://github.com/greenbone/go-project-template/pull/1 that it should be set higher ;) 

BUT it actually has nothing to do with the original theme from this PR wo you might as well merge the other one first...